modules/custom/az_person/config/install/core.entity_view_display.node.az_person.az_card.yml

```diff
@@ -103,10 +103,25 @@ third_party_settings:
         attributes: ''
         effect: none
         speed: fast
+    group_title_link:
+      children:
+        - smart_title
+      label: 'Title link'
+      parent_name: group_card_body
+      region: content
+      weight: 12
+      format_type: link
+      format_settings:
+        classes: ''
+        show_empty_fields: false
+        id: ''
+        target: custom_uri
+        custom_uri: '[node:az-canonical-url]'
+        target_attribute: default
   smart_title:
     enabled: true
     settings:
-      smart_title__link: true
+      smart_title__link: false
```

Comment:
To complete the change started with the commit suggestion above, the `smart_title__tag` should be set to `''` and `smart_title__classes` can be set to `{  }` in the lines below.